core/src/main/java/org/apache/accumulo/core/conf/ConfigurationDocGen.java

```diff
@@ -80,16 +80,28 @@ void prefixSection(Property prefix) {
         + prefix.getKey() + "*** | ");
     doc.print(prefix.isExperimental() ? "**Experimental.** " : "");
     doc.println(
-        (depr ? "**Deprecated.** " : "") + strike(sanitize(prefix.getDescription()), depr) + " |");
+        (depr
+            ? "**Deprecated since " + prefix.deprecatedSince() + ".** "
+                + (prefix.isReplaced() ? "**Replaced by " + "<a href=\"#"
+                    + prefix.replacedBy().getKey().replace(".", "_") + "prefix\">"
+                    + prefix.replacedBy() + "" + "</a>.** " : "")
+            : "**Available since " + prefix.availableSince() + ".** ")
+            + strike(sanitize(prefix.getDescription()), depr) + " |");
```

Comment:
This code looks good, but I think we want to show "Available since" regardless of if it's deprecated. The above logic will only show it if it is not deprecated.

Also, can you work in line breaks (`<br>`), so that "Available since", "Deprecated since", and "Replaced by" show up on separate lines, above the description?